public/js/controllers/traffic_graphs_controller.js

```diff
@@ -1301,6 +1353,18 @@ export class TrafficGraphsController {
             }
         });
 
+        // Draw sent graph with user-selected y-axis limit
```

Comment:
update on failed graph too.

Reply:
I'll open another PR for this. It's quite different from the others.